src/main/java/no/nav/arbeidsgiver/sykefravarsstatistikk/api/felles/bransjeprogram/BransjeEllerNæring.java

```diff
@@ -0,0 +1,37 @@
+package no.nav.arbeidsgiver.sykefravarsstatistikk.api.felles.bransjeprogram;
+
+import no.nav.arbeidsgiver.sykefravarsstatistikk.api.felles.Næring;
+import no.nav.arbeidsgiver.sykefravarsstatistikk.api.statistikk.Statistikkategori;
+
+import java.util.Optional;
+
+public class BransjeEllerNæring {
+    private final Optional<Bransje> bransje;
```

Comment:
Her klager Intellij. Begrunnelse:

> Optional was designed to provide a limited mechanism for library method return types in which a clear way to represent "no result" was needed.
> Using a field with the java.util.Optional type is also problematic if the class needs to be Serializable, as java.util.Optional is not serializable.

Reply:
Da bruker vi `Either<Næring, Bransje>` fra io.vavr som et bedre alternativ til `Optional` + en constructor for hver mulighet/type `Bransje` og `Næring`